document/CoreFormat.mdk

```diff
@@ -413,6 +413,44 @@ The `results` array shall be empty if the run that produced the `run` object did
 If the tool was run solely for the purpose of exporting rule metadata (see &sect;[#run-rules]),
 the `results` property shall be absent.
 
+### `toolNotifications` { #run-toolNotifications }
+
+A `run` object may contain a property named `toolNotifications`
+whose value is an array of zero or more `notification` objects (&sect;[#notification]).
+Each element of the array represents a runtime condition detected by the tool.
+The presence within this array of any `notification` object whose `level` property (&sect;[#notification-level]) is `error` shall mean
+that the run failed.
+
+~ IsoNote
+Depending on the nature of the error, a tool that encounters a runtime error might or might not be able to continue running.
+
+If the error occurs in the course of evaluating a rule, the tool might report the error in `toolNotifications`, disable the rule,
+and continue to execute the remaining rules.
+
+If the error occurs outside of the evaluation of a rule, the tool might report
```

Comment:
This was existing text. What I wanted to convey was that the tool might just plain crash before it has time to report the error. Of course, I said "either normally or abnormally", which sounds like it's ok for tools not to report the problem even if they _do_ have the chance to report.

How about replacing the last sentence with "If the tool exits abnormally, it might not have the opportunity to report the notification."?